### ISSUE TITLE
 Mark predicates

### ISSUE THREAD

**Issue by pagelj** [nilsreiter/CorefAnnotator#136]
It would be nice to be able to mark predicates in copula constructions and later on link them to their subject automatically. 

My idea would be to create normal entities for the predicates and mark mentions inside an entity with an additional attribute "predicate". It shouldn't be the whole entity which gets the attribute, as there might be later coreferent mentions not realized as a predicate.

However this will create a lot of singleton entities, since predicate entities will in fact not form coreference chains very frequently.

**Comment by nilsreiter** [comment]
Wouldn't it be better to somehow link the predicate to an existing mention?

**Comment by pagelj** [comment]
I think, we would need a new annotation view for that, since it would be a different type of link than a coreference link. Not sure if it is worth the trouble. As long as their is some sort of dependency parsing in the pipeline that gives the subject of a clause, it should be possible to do the linking automatically. But yes ideally a manually set link between predicate and subject would be optimal.

**Comment by nilsreiter** [comment]
Technically, this boils done to letting people add another text segment to mentions, which is feasible I think.

**Comment by melandresen** [comment]
We just submitted a paper to EADH were we extracted predicates based on manual coreference annotation and automatic dependency parsing. We tested this on two novels and it works quite well, meaning that you learn a lot about the respective characters. The results differed a lot depending on the parser used. As we did not have a manual annotation of predicates we could only evaluate for precision but not for recall.
